### ISSUE TITLE
Accessibility : `Publish your code` Dialog not accessible by NVDA screen reader.

### ISSUE THREAD

**Issue by marsonparulian** [exercism/website-copy#2210]
After clicking `Mark as complete` button, a `Publish your code..` dialog appears.

The dialog is not accessible by screen reader. 
I use combination of NVDA / magnifying tool. I have used all keys, but still not accessible. 
I have not try other screen readers since I don't have access to them.

This issue could prevent fully screen users to publish their solutions.
Thank you.
